src/css/aclmanagement.css

```diff
@@ -244,3 +243,20 @@ tr.acl-rule.edit-rule-row .form-control:not(.select-rule){
 .autocomplete-results-wrapper {
     z-index: 1000;
 }
+
+input.ng-invalid.ng-touched, select.ng-invalid.ng-touched, textarea.ng-invalid.ng-touched, form.ng-submitted .ng-invalid {
+    border: 2px solid #FA787E;
+}
+
+table td {
+    vertical-align: top !important;
+}
+
+.table-info, .table-info > td, .table-info > th {
+    background-color: var(--color-warning-light) !important;
+}
+
+.resource-tab .nav-link > span,
```

Comment:
There is no such thing as resource in the ACL view. I believe that the copy-pasted code was not properly altered to fit this view.